concrete/src/Page/Collection/Version/Version.php

```diff
@@ -261,6 +261,9 @@ public static function get($c, $cvID)
             case 'RECENT':
                 $q .= ' order by cvID desc limit 1';
                 break;
+            case 'RECENT_UNAPPROVED':
+                $q .= 'and (cvIsApproved = 0 or cvIsApproved IS NULL) order by cvID desc limit 1';
```

Comment:
A page is not approved also when its publish start time is in the future, or its publish end time is in the past.
So I'd add:
```sql
or (cvPublishDate is not null and cvPublishDate > ?) or (cvPublishEndDate is not null and cvPublishEndDate < ?)
```
and
```php
$v[] = $now;
$v[] = $now;
```

Reply:
A page that is scheduled has been approved by admin. It isnt published yet but it has been approved. In the database it will still contain cvIsApproved 1.

Adding it to UNAPPROVED makes no sense, thats why I also removed it from the retrieve versions (It was only added in october) 
With the change you added in october, pages that are scheduled would show up under both scheduled and unapproved. If a user wants to search unapproved pages, to review them, they shouldn't see scheduled pages. Pre commit - cecc6a01c717e6db928c2dfe430e94edff85e619 - you would only get the unapproved pages